apps/docs/content/guides/auth/third-party/firebase-auth.mdx

```diff
@@ -0,0 +1,254 @@
+---
+id: 'auth-third-party-firebase-auth'
+title: 'Firebase Auth'
+subtitle: 'Use Firebase Auth with your Supabase project'
+---
+
+<Admonition type="note">
+
+Creating a Firebase Auth third-party integration with your Supabase project is currently limited to a private alpha release. We are still trying to improve the developer experience around securing your project. Please reach out to [Supabase Support](https://supabase.help) if you're interested in having this enabled on your project.
+
+</Admonition>
+
+Firebase Auth can be used as a third-party authentication provider alongside Supabase Auth, or standalone, with your Supabase project.
+
+## Getting started
+
+1. First you need to add an integration to connect your Supabase project with your Firebase project. You will need to get the Project ID in the [Firebase Console](https://console.firebase.google.com/u/0/project/_/settings/general).
+2. Add a new Third-party Auth integration in your project's [Authentication settings](/dashboard/project/_/settings/auth).
+3. Create and attach restrictive RLS policies to all tables in your public schema, Storage and Realtime to prevent unauthorized access from unrelated Firebase projects.
```

Comment:
```suggestion
3. If you are using Third Party Auth in local development, create and attach restrictive RLS policies to all tables in your public schema, Storage and Realtime to prevent unauthorized access from unrelated Firebase projects.
```